### DISCUSSION TITLE
Connect aws postgres db to the vercel

### DISCUSSION THREAD

**Discussion by rldnd1818** [vercel/vercel#5139] [Help] [answered]
i'm trying to connect to vercel with aws postgres db
I've created an instance of free tier rds and i got endpoint
so this is my server code

 ```
const connection = await createConnection({
type: 'postgres',
url: process.env.DATABASE_URL,
logging: false,
synchronize: false,

entities: [
  User,
],
});
```
so my main question is how to create url

I've tried DATABASE_URL=postgresql://user:password@my aws endpoint:5432/my aws database name
and not worked vercel error said
ERROR : Task timed out after 10.01 seconds
What did I do wrong?

**Comment by styfle** [comment, marked as answer]
You'll need to add the `DATABASE_URL` to both Preview and Production environments if you plan to use the same database for both or you can assign different values.

Learn more: https://vercel.com/docs/build-step#environment-variables

You might need to allow connections for all IP Addresses too.

Learn more: https://vercel.com/knowledge/how-to-allowlist-deployment-ip-address
